ui/src/styles/globals.css

```diff
@@ -59,6 +59,13 @@
     .custom-table-tiptap-editor td {
         @apply border border-teal-500 px-1 py-0 lg:px-2;
     }
+
+    .visualization-box-title {
```

Comment:
did you consider adding

https://github.com/tailwindlabs/tailwindcss-line-clamp

Reply:
I've been looking for something similar but didn't find this. Thanks

Reply:
added tailwindcss line-clamp plugin